src/server/game/Spells/SpellEffects.cpp

```diff
@@ -5613,17 +5613,16 @@ void Spell::EffectCreateAreaTrigger(SpellEffIndex /*effIndex*/)
     if (effectHandleMode != SPELL_EFFECT_HANDLE_HIT)
         return;
 
+    Unit* target = unitTarget != nullptr ? unitTarget : GetCaster();
```

Comment:
unitTarget always null here

you must have a valid destTarget here, otherwise something went wrong